### DISCUSSION TITLE
Toggle Transponder on

### DISCUSSION THREAD

**Discussion by JoergM-D** [nguyenquyhy/Flight-Tracker-StreamDeck#260] [Q&A]
Hello community,
how can can I toggle inside the C172 the transponder (XPNDR) to on ?

Cheers Jörg

**Comment by rmroc451** [comment]
Id fire in an issue, requesting the following mobiflight events. I pulled this from my airmanager implementation:

`fs2020_event("MobiFlight.TransponderOFF")`
`fs2020_event("MobiFlight.TransponderSTBY")`
`fs2020_event("MobiFlight.TransponderTST")`
`fs2020_event("MobiFlight.TransponderON")`
`fs2020_event("MobiFlight.TransponderALT")`

these all work on my knobster implementation, so no reason we couldn't have similar events added to the enum in the streamdeck app.

**Comment by JoergM-D** [comment]
Thanks, I got it now running with Lorby's aao.
